plugin/inlay_hint.py

```diff
@@ -142,12 +143,19 @@ def format_inlay_hint_label(inlay_hint: InlayHint, session: Session, phantom_uui
             })
             result += f'<a href="{inlay_hint_click_command}">'
         instruction_text = '\nDouble-click to insert' if has_text_edits else ""
-        result += f'<span title="{(tooltip + instruction_text).strip()}">{html.escape(label)}</span>'
+        truncated = len(label) > truncate_limit
+        truncated_label = label[:truncate_limit] + '…' if truncated else label
+        tooltip_label = "".join(label_part['value'] for label_part in label)
```

Comment:
Typechecker like LSP-pyright should tell you that there is an error here `label` is a string so `label_part['value']` doesn't exist in this case.